src/liquid/components/ld-toggle/readme.md

```diff
@@ -0,0 +1,186 @@
+  key: Toggle
+  parent: Components
+layout: layout.njk
+title: Toggle
+permalink: components/ld-toggle/
+---
+
+# ld-toggle
+
+A toggle allows the user to select and deselect single values.
+
+This component can be used in conjunction with the [`ld-label`](components/ld-label/) and the [`ld-input-message`](components/ld-input-message/) component.
+
+---
+
+## Examples
+
+### Primary
+
+{% example %}
+<ld-toggle></ld-toggle>
+
+<!-- CSS component -->
+
+<button type="button" class="ld-toggle">
+  <span class="ld-toggle__knob"></span>
+</div>
+{% endexample %}
+
+### Large
+
+{% example %}
+<ld-toggle mode="large"></ld-toggle>
+
+<!-- CSS component -->
+
+<button type="button" class="ld-toggle ld-toggle--large">
+  <span class="ld-toggle__knob"></span>
+</button>
+{% endexample %}
+
+### Disabled
+
+{% example %}
+<ld-toggle disabled></ld-toggle>
+
+<ld-toggle disabled checked></ld-toggle>
+
+<!-- CSS component -->
+
+<button type="button" class="ld-toggle" disabled>
+  <span class="ld-toggle__knob"></span>
+</button>
+
+<button type="button" class="ld-toggle" disabled aria-checked="true">
+  <span class="ld-toggle__knob"></span>
+</button>
+{% endexample %}
+
+**If you want the toggle to stay focusable** even if it is disabled, use `aria-disabled` in place of `disabled`:
+
+{% example %}
+<ld-toggle aria-disabled="true"></ld-toggle>
+
+<ld-toggle aria-disabled="true" checked></ld-toggle>
+
+<!-- CSS component -->
+
+<button type="button" class="ld-toggle" aria-disabled="true" id="focusable-disabled-toggle-1">
+  <span class="ld-toggle__knob"></span>
+</button>
+
+<button type="button" class="ld-toggle" aria-disabled="true" aria-checked="true" id="focusable-disabled-toggle-2">
+  <span class="ld-toggle__knob"></span>
+</button>
+
+<!-- Example code for input prevention on aria-disabled toggle elements -->
+<script>
+  const inputs = document.querySelectorAll('#focusable-disabled-toggle-1, #focusable-disabled-toggle-2')
+  Array.from(inputs).forEach(input => {
+    input.addEventListener('click', (ev) => {
+      ev.preventDefault()
+    })
+  })
+</script>
+
+{% endexample %}
+
+> **Note:** When `aria-disabled` is applied on the toggle, the component will try to prevent user interaction using an internal click event handler, calling `preventDefault()` on the click event. With the CSS component version on the other hand, you will need to take care of preventing the default behaviour of the toggle yourself.
+
+### With label
+
+{% example %}
+<ld-label position="right" size="m">
+I have read the terms of service.\*
+<ld-toggle></ld-toggle>
+</ld-label>
+
+<!-- CSS component -->
+
+<label class="ld-label ld-label--right ld-label--m">
+  I have read the terms of service.*
+  <button type="button" class="ld-toggle" required>
+    <span class="ld-toggle__knob"></span>
+  </button>
+</label>
+{% endexample %}
+
+Please refer to the [ld-label](components/ld-label/) docs for more information on the label component.
+
+### With label and input message
```

Comment:
I noticed a visual difference between the Web Component and the CSS Component implementations in this example. Can you check, please?

Reply:
It's due to the `min-width` property of the child selector of the `ld-label` element (`.ld-label--left>:not(ld-input-message):not(.ld-input-message), .ld-label--right>:not(ld-input-message):not(.ld-input-message)`). In case of the `ld-checkbox` you overrode this explicitly. Wouldn't it be cleaner not to handle this problem inside the `ld-toggle`, but rather in `ld-label`?

Reply:
Would you like to dig into this and make the change in ld-label?

Reply:
As with the new `<input type="checkbox />` implementation this happened for both, the web- and the css-component, I fixed this the same way you did with `ld-checkbox` now. But I'll put a refactoring on my to-do list.